### PR TITLE
Extract fence_vbox

### PR DESCRIPTION
**Pull request by petertix**
- Online-Help #33
 - Created a file for Setting up Power Control
 - Added the link to the file in two places.

Signed-off-by: Peter Robinson <peter.timothy.robinson@gmail.com>

### CODE REVIEW

File: docs/Contributor_Docs/cd_Setting_Up_Power_Control.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    # Enter your computer password
+```
+
+**Note** There is currently a bug that causes the fake_ipmi_vbox.py script to fail on the first run; the process will hang indefinitely. 
+
+To fix this, hit `ctrl+Z` to suspend the process. 
+
+Next, run `ps aux | grep "fake_ipmi"` to get the process number and kill the process by executing, `kill -9 <processId>`. 
+
+Once the process has been destroyed, run the script a second time, using the same information and the script should succeed.
```
Comment (jgrund) [inline docs/Contributor_Docs/cd_Setting_Up_Power_Control.md]: This (running the script a second time) is actually not needed.

File: docs/Contributor_Docs/cd_Setting_Up_Power_Control.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Navigate to `Configuration->Power Control`. Add the following entries for each server:
+
+|Server|	PDU |
+|--------|-------|
+| mds1.lfs.local|mds1 |
+| mds2.lfs.local|mds2 |
+| oss1.lfs.local|oss1 |
+| oss2.lfs.local|oss2 |
+| | |
```
Comment (jgrund) [inline docs/Contributor_Docs/cd_Setting_Up_Power_Control.md]: Empty row here

Comment (petertix) [reply docs/Contributor_Docs/cd_Setting_Up_Power_Control.md]: Removed empty row

File: docs/Contributor_Docs/cd_Setting_Up_Power_Control.md (hunk 1 above)
Comment (petertix) [reply docs/Contributor_Docs/cd_Setting_Up_Power_Control.md]: Removed:
run the script a second time, using the same information and the script should succeed.

File: docs/Contributor_Docs/cd_Setting_Up_Power_Control.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    # Enter your computer password
+```
+
+**Note** There is currently a bug that causes the fake_ipmi_vbox.py script to fail on the first run; the process will hang indefinitely. 
+
+To fix this, hit `ctrl+Z` to suspend the process. 
+
+Next, run `ps aux | grep "fake_ipmi"` to get the process number and kill the process by executing, `kill -9 <processId>`. 
+
+Once the process has been destroyed,
```
Comment (tanabarr) [inline docs/Contributor_Docs/cd_Setting_Up_Power_Control.md]: do you have to then rerun the script?

Comment (jgrund) [reply docs/Contributor_Docs/cd_Setting_Up_Power_Control.md]: No.

**Pull request merged**